packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -195,6 +201,24 @@ export class KlevuQuicksearch {
     composed: true,
   })
   klevuData!: EventEmitter<KlevuQuicksearchDataEvent>
+  /**
+   * When user clicks search button. Returns the search term.
+   * This event is emitted when there is no url matched for redirects
+   */
+  @Event({
+    composed: true,
+  })
+  klevuSearch!: EventEmitter<string>
+
+  /**
+   * Will be emitted when there is a url match for redirects.
+   * You can override the default behaviour of redirects by
+   * preventing default of this event
+   */
+  @Event({
+    composed: true,
+  })
+  klevuRedirect!: EventEmitter<string>
```

Comment:
We should send the whole object of redirect instead of just the url.

Reply:
changed type.